mqtt/mqtt-util/src/client_io.rs

```diff
@@ -309,41 +309,30 @@ pub enum Credentials {
 
 #[derive(Debug, Clone, PartialEq, Deserialize)]
 pub struct CredentialProviderSettings {
-    #[serde(rename = "iotedge_iothubhostname")]
```

Comment:
this annotation no longer needed, since we do env var mappings in `BridgeEnvironment`